src/environments/environment.prod.ts

```diff
@@ -6,4 +6,9 @@
 
 export const environment = {
   production: true,
+  APP_ETH_PARSER_URL: 'https://ethparser.herokuapp.com',
```

Comment:
need to use sttings, this config will not work on heorku

Reply:
fixed